specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/outputs.json

```diff
@@ -490,6 +490,31 @@
         "type"
       ]
     },
+    "RawOutputDatasource": {
+      "description": "Describes a raw output data source.",
+      "x-ms-discriminator-value": "Raw",
+      "allOf": [
+        {
+          "$ref": "#/definitions/OutputDataSource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties that are associated with a raw output.",
+          "$ref": "#/definitions/RawOutputDatasourceProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RawOutputDatasourceProperties": {
+      "description": "The properties that are associated with a raw output.",
+      "properties": {
+        "payloadUri": {
+          "description": "The SAS URL to a blob where the output should be written. If this property is not set, output data will be embedded in the result response. The SAS URL needs to have write permission.",
```

Comment:
NIT: I think we decided not to embed the data result into response. In the case that payloadUri is not specified, we will write it into a temporary storage and give the SAS url to that. You can double check with Ken. I also believe there was a comment thread about it in Ken's PR introducing the API.